From d9cc98e262a351b86e10a1a9b0af7dc59df34bdd Mon Sep 17 00:00:00 2001
From: Timon <timon@skerutsch.de>
Date: Fri, 5 Feb 2021 22:20:41 +0100
Subject: [PATCH] Added CM4 hwver

The two hardware revision codes so far seen used for the Raspberry Pi CM4.
---
 rpihw.c | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)

diff --git a/rpihw.c b/rpihw.c
index adb7efe..d9b7c80 100644
--- a/rpihw.c
+++ b/rpihw.c
@@ -121,6 +121,23 @@ static const rpi_hw_t rpi_hw_info[] = {
         .videocore_base = VIDEOCORE_BASE_RPI2,
         .desc = "Pi 4 Model B - 2GB v1.4"
     },
+    //	
+    // Compute Module 4
+    //
+    {
+        .hwver = 0xa03140,
+        .type = RPI_HWVER_TYPE_PI4,
+        .periph_base = PERIPH_BASE_RPI4,
+        .videocore_base = VIDEOCORE_BASE_RPI2,
+        .desc = "Compute Module 4 v1.0"
+    },
+    {
+        .hwver = 0xb03140,
+        .type = RPI_HWVER_TYPE_PI4,
+        .periph_base = PERIPH_BASE_RPI4,
+        .videocore_base = VIDEOCORE_BASE_RPI2,
+        .desc = "Compute Module 4 v1.0"
+    },
     //
     // Model B Rev 1.0
     //
